### PR TITLE
GW-4039 fixed the error in codes

### PR DESCRIPTION
**Pull request by sacrrie**
there's a function didn't bind correctly so that so it prevents the pagination to work

### CODE REVIEW

File: src/client/js/components/Admin/Security/ShareLinkSetting.jsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ class ShareLinkSetting extends React.Component {
       shareLinks, shareLinksActivePage, totalshareLinks, shareLinksPagingLimit,
     } = adminGeneralSecurityContainer.state;
 
-    function pager() {
+    const pager = () => {
```
Comment (yuki-takei) [inline src/client/js/components/Admin/Security/ShareLinkSetting.jsx:90]: `const Pager = () => {`
に変えて

File: src/client/js/components/Admin/Security/ShareLinkSetting.jsx [hunk 2]
```diff
@@ -115,7 +115,7 @@ class ShareLinkSetting extends React.Component {
           </button>
           <h2 className="alert-anchor border-bottom">{t('share_links.share_link_management')}</h2>
         </div>
-        {pager}
+        {pager()}
```
Comment (yuki-takei) [inline src/client/js/components/Admin/Security/ShareLinkSetting.jsx:118]: こっちは `<Pager />`

**Review by yuki-takei — CHANGES_REQUESTED** [review]

File: src/client/js/components/Admin/Security/ShareLinkSetting.jsx (hunk 1 above)
Comment (sacrrie) [reply src/client/js/components/Admin/Security/ShareLinkSetting.jsx:90]: はい

File: src/client/js/components/Admin/Security/ShareLinkSetting.jsx (hunk 2 above)
Comment (sacrrie) [reply src/client/js/components/Admin/Security/ShareLinkSetting.jsx:118]: 改正しました

**Pull request merged**